docs/content/stack/configuration-variables.md

```diff
@@ -41,7 +41,7 @@ The default amount of disk space allocated to Docksal's VirtualBox machine, in m
 
 `Default: 0`
 
-Applicable to macOS or Windows.  
```

Comment:
Looks like your IDE is configured to remove extra spaces. Spaces may be important. Like this  double space that breaks the line. Please revert the irrelevant changes that were pushed along in this PR.